### ISSUE TITLE
Improve error message when adding a wrong localhost URL

### ISSUE THREAD

**Issue by curquiza** [meilisearch/meilisearch#4811]
Following this issue: https://github.com/meilisearch/meilisearch/issues/4697

Improve the error message when an URL that lacks the protocol/scheme part is provided to Meilisearch

Currently we return: "Failed to connect to Meilisearch: No connection adapters were found for 'localhost:7700/version"

We want to be more explicit that the protocol/scheme is missing

**Comment by curquiza** [comment]
@flevi29, hum, i'm not sure, why do you say this?

**Comment by flevi29** [comment]
Just going after the error message, [it looks like a python thing](https://stackoverflow.com/questions/15115328/python-requests-no-connection-adapters).

**Comment by curquiza** [comment]
Discussed with @ManyTheFish and this is not only a problem to fix on Python's side

In the error message, we may want to add an additional information saying that it’s related to `--http-addr`  or `MEILI_HTTP_ADDR`

**Comment by irevoire** [comment]
Yes, and since the SDK couldn’t reach Meilisearch, it definitely cannot be a Meilisearch error message
